setup.cfg

```diff
@@ -22,6 +22,8 @@ packages =
     spotify_tensorflow
 
 [extras]
+luigi =
+    luigi>=2.3.3,<3.0.0
 tfx =
```

Comment:
do we also want to change it to tfdv? since it's only tfdv now.

Reply:
sure, good idea. will do in a separate pr.